src/main/scala/org/broadinstitute/dsde/firecloud/core/ProfileClient.scala

```diff
@@ -87,8 +89,10 @@ class ProfileClientActor(requestContext: RequestContext) extends Actor with Fire
 
       profileResponse pipeTo context.parent
 
-    case UpdateNIHLink(userInfo: UserInfo, nihLink: NIHLink) =>
+    case UpdateNIHLinkAndSyncSelf(userInfo: UserInfo, nihLink: NIHLink) =>
       val parent = context.parent
+      // Complete syncWhitelist and ignore as neither success nor failure are useful to the client
+      ask(self, ProfileClient.SyncSelf(userInfo))(1.minute)
```

Comment:
does this wait for the ask to complete? i.e. does this need Await.ready? I haven't tried it, but just from reading the code I wonder ...